app/controllers/user.controller.js

```diff
@@ -10,19 +10,31 @@ exports.create = async (req, res) => {
   if (req.body.firstName === undefined) {
     const error = new Error("First name cannot be empty for user!");
     error.statusCode = 400;
-    throw error;
+    res.status(400).send({
+      message:
+        error.message || "Some error occurred while creating user.",
```

Comment:
don't you set the `Error` message just a few lines above on line 11?  Why would it be missing or empty?